shared/common-adapters/icon.desktop.tsx

```diff
@@ -163,13 +163,14 @@ class Icon extends Component<Props, void> {
   }
 }
 
-const imgName = (type: IconType, ext: string, mult: number, prefix?: string, postfix?: string) =>
```

Comment:
intentionally removing typing for icon names?

Reply:
yeah its fine. its not exported and only called internally by iconTypeToSrcSet